justfile

```diff
@@ -49,20 +49,18 @@ py-build:
 py-format:
     black --config rerun_py/pyproject.toml {{py_folders}}
     blackdoc {{py_folders}}
-    isort {{py_folders}}
     pyupgrade --py37-plus `find rerun_py/rerun/ -name "*.py" -type f`
+    ruff --fix --config rerun_py/pyproject.toml  {{py_folders}}
 
 # Check that all the requirements.txt files for all the examples are correct
 py-requirements:
     find examples/python/ -name main.py | xargs -I _ sh -c 'cd $(dirname _) && echo $(pwd) && pip-missing-reqs . || exit 255'
 
 # Run linting
-py-lint: py-requirements
+py-lint:
     black --check --config rerun_py/pyproject.toml --diff {{py_folders}}
     blackdoc --check {{py_folders}}
-    isort --check {{py_folders}}
-    mypy --no-warn-unused-ignore
```

Comment:
does `ruff` replace `mypy` too? Does it really do type-checking?

Reply:
Sadly no -- I mentally mixed up mypy and flake8 -- will restore that. As much as I want to delete it.

Reply:
Restored and updated to version 1.0.1